ChangeLog.md

```diff
@@ -21,6 +21,9 @@ repository history](https://github.com/ddclient/ddclient/commits/master).
     minute.
   * The `pid` option is now ignored when ddclient is not daemonized.
   * ddclient now gracefully exits when interrupted by Ctrl-C.
+  * The way ddclient chooses the default for the `use` option has changed.
```

Comment:
I think it would be nice to mention how it is chosen now.

Reply:
I intentionally omitted that because I want users to explicitly set `use`. I view this feature as "the config is broken, but let's try to do something useful anyway". Should we change the documentation for `use` to make this behavior official?

Reply:
Yeah, you're right they should explicitly set it.